recipes/nsimd/2.x/test_package/test_package.cpp

```diff
@@ -0,0 +1,5 @@
+#include <nsimd/nsimd-all.hpp>
+
+int main() {
+    nsimd::pack<float> v;
```

Comment:
NIT: if possible, calling some method would be nice (like computing sum of two vectors?)
but instantiation is good as well

Reply:
True.  I've now added some sample code from the library's docs.